packages/apple-pay-integration/src/apple-pay-session-factory.ts

```diff
@@ -8,7 +8,7 @@ function isApplePayWindow(window: Window): window is ApplePayWindow {
 
 export function assertApplePayWindow(window: Window): asserts window is ApplePayWindow {
     if (!isApplePayWindow(window)) {
-        throw new Error('Apple pay is not supported');
+        throw new Error('ApplePaySDK is missing in the window');
```

Comment:
It might be better to revert this change, since it will be shown to our shoppers, who is not responsible for dev part.

Reply:
This message will only be visible in the console and will not be visible to our customers